src/common/components/layouts/pages/inspect/pane/Pane.scss

```diff
@@ -0,0 +1,5 @@
+.pane {
+    grid-area: pane;
+    min-width: 0 !important;
```

Comment:
bad practise to use important, please try to use  .classThatIsParentForCurrentElement .pane , it can be even the main css from app body